news/204.bugfix

```diff
@@ -0,0 +1,2 @@
+Pass messages from translate method to get them translated for folder_contents view
```

Comment:
"to pass to" instead of "to pass from", and technically translate is a function, not a method.
=> Pass messages to the translate function to get them translated for folder_contents view